packages/utils/classes.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, test } from "bun:test";
+import { classes } from "./classes";
+
+test("test_with_empty_arguments", () => {
+  const input: string[] = [];
+
+  const expected = "";
+
+  expect(classes(...input)).toBe(expected);
+});
+
+test("test_with_valid_argument", () => {
+  const input: string[] = ["bg-red-500"];
+
+  const expected = "bg-red-500";
+
+  expect(classes(...input)).toBe(expected);
+});
+
+test("test_with_multiple_valid_arguments", () => {
+  const input: string[] = ["bg-red-500", "text-white"];
+
+  const expected = "bg-red-500 text-white";
+
+  expect(classes(...input)).toBe(expected);
+});
+
+test("test_with_dynamic_argument", () => {
+  const darkMode = Math.random() > 0.5 ? "dark" : "light";
+  const input: string[] = ["bg-red-500", darkMode];
+
+  const expected = `bg-red-500 ${darkMode}`;
+
+  expect(classes(...input)).toBe(expected);
+});
+
+test("test_with_non_string_argument", () => {
+  const nonStringArgument = true;
+  const input: Array<string | boolean> = ["bg-red-500", nonStringArgument];
+
+  const expected = "bg-red-500";
+
+  expect(classes(...(input as string[]))).toBe(expected);
+});
+
+test("test_with_null_and_undefined_arguments", () => {
+  const input: Array<string | null | undefined> = ["bg-red-500", null, undefined];
+
+  const expected = "bg-red-500";
+
+  expect(classes(...(input as string[]))).toBe(expected);
+});
+
+test("test_with_empty_string_argument", () => {
+  const input: string[] = ["bg-red-500", ""];
+
+  const expected = "bg-red-500";
+
+  expect(classes(...input)).toBe(expected);
+});
+
+test("test_with_mixed_arguments", () => {
+  const input: Array<string | null | undefined | number> = ["bg-red-500", "", null, undefined, "text-white", 42];
+
+  const expected = "bg-red-500 text-white";
+
+  expect(classes(...(input as string[]))).toBe(expected);
+});
+
+test("test_with_only_invalid_arguments", () => {
+  const input: Array<string | null | undefined | boolean | number> = [null, undefined, true, false, 42];
+
+  const expected = "";
+
+  expect(classes(...(input as string[]))).toBe(expected);
+});
+
+test("test_with_duplicate_arguments", () => {
```

Comment:
I missed a test case for multiple tokens on the same string:
```js
classes("bg-red-500 w-4", "bg-red-500 px-3"); // bg-red-500 w-4 px-3
```

Reply:
Added in: https://github.com/podcodar/webapp/pull/175/commits/e24fcc25615c1dd8ffa91cf5fdc33df03d947ad9